client/src/components/SettingsPanel.tsx

```diff
@@ -33,46 +67,178 @@ interface IWalletItem {
 
 export const SettingsPanel = ({
 	onChangeName,
-	onSubmitUrl
+	onSubmitUrl,
+	onChangeAvatar,
+	onSendLocation
 }: ISettingsPanelProps) => {
-	const classes = useStyles();
 	let walletItems: IWalletItem[] = [];
 	const [items, setItems] = useState(walletItems);
-
+	const [isWalletLoaded, setIsWalletLoaded] = useState(false);
 	const { isLoggedIn, accountId } = useContext(AuthContext);
+	const [rooms, setRooms] = useState<IRoomData[]>([]);
+
+	const [activeAvatar, setActiveAvatar] = useState("");
+	const avatars = [{
```

Comment:
should declare static objects not dependent on props/state outside the component

Reply:
what is a static objet and where is the one dependent on props/state outside the settingspanel component?